src/lib/list/list.ts

```diff
@@ -156,7 +156,7 @@ export class MdListItem extends _MdListItemMixinBase implements AfterContentInit
   }
 
   /** Whether this list item should show a ripple effect when clicked.  */
-  isRippleEnabled() {
+  _isRippleEnabled() {
```

Comment:
Perhaps rename this to `_rippleDisabled` so it matches the mixin name?

Reply:
Yeah that might be easier for the template as well.

Reply:
Done. Still named it `_isRippleDisabled`.